Tests/MapboxMapsTests/Gestures/GestureHandlers/DoubleTapToZoomInGestureHandlerTests.swift

```diff
@@ -49,6 +49,13 @@ final class DoubleTapToZoomInGestureHandlerTests: XCTestCase {
         XCTAssertEqual(cameraAnimationsManager.easeToStub.parameters.first?.camera, CameraOptions(zoom: mapboxMap.cameraState.zoom + 1))
         XCTAssertEqual(cameraAnimationsManager.easeToStub.parameters.first?.duration, 0.3)
         XCTAssertEqual(cameraAnimationsManager.easeToStub.parameters.first?.curve, .easeOut)
-        XCTAssertNil(cameraAnimationsManager.easeToStub.parameters.first?.completion)
+        XCTAssertNotNil(cameraAnimationsManager.easeToStub.parameters.first?.completion)
+        XCTAssertEqual(delegate.gestureEndedStub.parameters.first?.gestureType, .doubleTapToZoomIn)
+        do {
+            let willAnimate = try XCTUnwrap(delegate.gestureEndedStub.parameters.first?.willAnimate)
+            XCTAssertTrue(willAnimate)
+        } catch {
+            XCTFail(error.localizedDescription)
+        }
```

Comment:
Mark this test case with `throws` and then change these lines to

```suggestion
            let willAnimate = try XCTUnwrap(delegate.gestureEndedStub.parameters.first?.willAnimate)
            XCTAssertTrue(willAnimate)
```

Reply:
This feedback applies throughout this PR since there are multiple places with this same pattern.